### PR TITLE
Fixed #571 Added Service Worker

### LINKED ISSUE #571: Adding PWA Features to the Site
### Expected results
Should Load Faster
### Actual results
Loads Very Slowly
### Steps to reproduce the issue
Add Service Workers and Cashing
## Would you like to work on this
Yes.

### PR DESCRIPTION
**Pull request by niranjannitesh**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request
Added Service Worker to the website to improve site speed.
Preview URL: https://niteshkumarniranjan.github.io/gci17.fossasia.org/

- Fixes #571

### CODE REVIEW

**Review by harshit98 — APPROVED** [review]
Good work @niteshkumarniranjan 💯

**Comment by harshit98** [comment]
It would be better if you can provide speed comparison screenshots as well before and after.

File: sw.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        'css/stylesheet.css',
+        'css/lightbox.min.css',
+        'css/theme-red.css',
+        'css/animate.css',
+        'css/hover.css',
+        'css/custom.css',
+        'css/blog.css',
+        'css/hover-media-links.css',
+        'js/jquery.min.js',
+		'js/bootstrap.min.js',
```
Comment (vaibhavsingh97) [inline sw.js:20]: improper indentation

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Comment by niranjannitesh** [comment]
# I have a BSNL 3G Connection

## Before Service Worker
Finish Time:  3.0 mins
Data Transferred: 7.7MB
screenshot 101

## After Service Worker
Finish Time:  59 sec.
Data Transferred: 5.6MB
screenshot 100

**Comment by niranjannitesh** [comment]
@vaibhavsingh97

**Comment by harshit98** [comment]
Looks good to me but I'm closing this PR. Create a fresh PR and make sure only 1 commit is there.

**Comment by niranjannitesh** [comment]
okay!

**Comment by harshit98** [comment]
@niteshkumarniranjan Good. Kindly check my comments on the task as well which is related to this pull request.

**Pull request closed without merging**